### ISSUE TITLE
Release binaries: include virtio-fs

### ISSUE THREAD

**Issue by jcrowthe** [kata-containers/runtime#2126]
**Describe the solution you'd like**

In the release binaries, `/opt/kata/kata-qemu` is a bash script containing the following:

```
#!/bin/bash
/opt/kata/bin/kata-runtime --kata-config "/opt/kata/share/defaults/kata-containers/configuration-qemu.toml" $@
```
This is a request to include `/opt/kata/kata-qemu-virtiofs` as well, with contents:

```
#!/bin/bash
/opt/kata/bin/kata-runtime --kata-config "/opt/kata/share/defaults/kata-containers/configuration-qemu-virtiofs.toml" $@
```

**Comment by egernst** [comment]
@jcrowthe this should be available now for 1.9 release of Kata.  PTAL, and close if this meets your needs!

**Comment by jcrowthe** [comment]
Fixed, thanks!
